src/app/containers/ArticleMain/index.jsx

```diff
@@ -17,15 +18,23 @@ const componentsToRender = {
   timestamp,
 };
 
+const avEnabledComment = (
+  // eslint-disable-next-line react/no-danger
+  <div dangerouslySetInnerHTML={{ __html: '<!-- av-enabled -->' }} />
```

Comment:
Could this be moved into the `DialContextProvider`? Have it comment about all dials it has, rather than throughout the app?

Reply:
Yeah it could, but this comment is just to demonstrate that the dial is working as intended. This code will be deleted once https://github.com/bbc/simorgh/pull/2151 is adapted to use the dial.